index.js

```diff
@@ -0,0 +1,35 @@
+const app = Vue.createApp({
+  data() {
+    return {
+      name: "John Cena",
+      address: "street no. 123, Shibuya, Japan",
+      dob: "2000-09-11",
+      age: null,
+      isValid: false,
+      showValidation: false,
```

Comment:
showValidation? what do you mean by showValidation, use proper variable name @TusharKumar7